init.sql

```diff
@@ -0,0 +1,31 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TABLE Rating (
```

Comment:
It would be good if you review the database table and column naming convention. It's not so common to see it using capital letter at the beginning.

Reply:
Done